jetty-server/src/main/java/org/eclipse/jetty/server/HttpOutput.java

```diff
@@ -908,6 +908,22 @@ public void setBufferSize(int size)
         _commitSize = size;
     }
 
+    public void onFlushed(long bytes) throws IOException
+    {
```

Comment:
probably should comment that the bytes flushed here could be after compression and thus less than the bytes written